android_mobile/src/main/java/com/alexstyl/specialdates/EasyPreferences.kt

```diff
@@ -0,0 +1,99 @@
+package com.alexstyl.specialdates
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.content.res.Resources
+import android.preference.PreferenceManager
+import android.support.annotation.BoolRes
+import android.support.annotation.StringRes
+import android.support.v4.util.Pair
+
+class EasyPreferences private constructor(private val prefs: SharedPreferences, private val res: Resources) {
+
+    private fun key(key: Int): String {
+        return res.getString(key)
+    }
+
+    fun getBoolean(@StringRes bool: Int, defValue: Boolean): Boolean {
```

Comment:
https://developer.android.com/reference/android/support/annotation/BoolRes I'd use this here since it's a boolean res (and on the others)